templates/unattended-upgrades.j2

```diff
@@ -30,25 +30,39 @@ APT::Periodic::RandomSleep "{{unattended_random_sleep}}";
 
 
 // Overrides for `50unattended-upgrades`. ////////////////////////////////////
-
+{% if unattended_origins_patterns is defined or __unattended_origins_patterns is defined %}
 // Unattended-Upgrade::Origins-Pattern controls which packages are
 // upgraded.
 Unattended-Upgrade::Origins-Pattern {
 {% if unattended_origins_patterns is defined %}
-  {% for origin in unattended_origins_patterns %}
-    "{{ origin }}";
-  {% endfor %}
+{% for pattern in unattended_origins_patterns %}
+  "{{ pattern }}";
+{% endfor %}
 {% else %}
-  {% for origin in __unattended_origins_patterns %}
-   "{{ origin }}";
-  {% endfor %}
+{% for pattern in __unattended_origins_patterns %}
+  "{{ pattern }}";
+{% endfor %}
 {% endif %}
 };
-
+{% endif %}
+{% if unattended_allowed_origins is defined or __unattended_allowed_origins is defined %}
+// Automatically upgrade packages from these (origin:archive) pairs
+Unattended-Upgrade::Allowed-Origins {
+{% if unattended_allowed_origins is defined %}
+{% for origin in unattended_allowed_origins %}
+  "{{ origin }}";
+{% endfor %}
+{% else %}
+{% for origin in __unattended_allowed_origins %}
+  "{{ origin }}";
+{% endfor %}
+{% endif %}
+};
+{% endif %}
 // List of packages to not update (regexp are supported)
 Unattended-Upgrade::Package-Blacklist {
 {% for package in unattended_package_blacklist %}
-      "{{package}}";
+  "{{package}}";
```

Comment:
2 last teeny changes here (line 65), and line 147. 

Since blacklist and dpkg options aren't part of our changes, lets preserve the original indentation, so the change only contains our intended modifications.